File: platforms/kotlin/examples/scene/scene/list_passes.kt
import org.fragmentcolor.*

val mesh = Mesh()
mesh.addVertex( Vertex.pbr(listOf(0.0f, 0.5f, 0.0f)).set("uv0", floatArrayOf(0.5f, 1.0f)), )
val scene = Scene()
scene.add(Model(mesh, Material.pbr()))

// Compose, don't clear: keep whatever the previous pass drew.
for (pass in scene.listPasses()) {
    pass.loadPrevious()
}

File: platforms/web/examples/scene/scene/list_passes.js
import { Material, Mesh, Model, Scene, Vertex } from "fragmentcolor";

const mesh = new Mesh();
mesh.addVertex( Vertex.pbr([0.0, 0.5, 0.0]).set("uv0", [0.5, 1.0]), );
const scene = new Scene();
scene.add(new Model(mesh, Material.pbr()));

// Compose, don't clear: keep whatever the previous pass drew.
for (const pass of scene.listPasses()) { pass.loadPrevious(); };